### PR TITLE
Adding dependsOn field to the artifacts

### PR DESCRIPTION
**Pull request by Skarlso**
### Description

Profiles part of https://github.com/weaveworks/profiles/issues/214

### Checklist
- [ ] Added tests that cover your change
- [ ] Added/modified documentation as required (such as the `README.md` (diagrams, usage, roadmap etc), and anything in `examples/`)
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -9,3 +9,9 @@ configMapGenerator:
 - files:
   - controller_manager_config.yaml
   name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: weaveworks/profiles-controller
+  newName: localhost:5000/profiles-controller
+  newTag: latest
```
Comment (Callisto13) [inline config/manager/kustomization.yaml:17]: is this that thing we said we would remember to not commit?

**Review by Callisto13 — APPROVED** [review]

File: config/manager/kustomization.yaml (hunk 1 above)
Comment (Skarlso) [reply config/manager/kustomization.yaml:17]: Grrrr. We need to ignore this then force push it.

**Comment by Skarlso** [comment]
@aclevername Yeah the force adding after ignoring doesn't work. :( Once it's tracked, it's tracked. No matter if it's ignored. We'll have to come up with something for this. :D

**Comment by Skarlso** [comment]
We could run `git update-index --assume-unchanged [<file> ...]` but that's for local only.
Another way would be to somehow have a `.modified` file of this that we can ignore and that the tests are using.

**Pull request merged**